VitaliiLaputko/src/test/java/ua/artcode/week1/FlashCardListTest.java

```diff
@@ -0,0 +1,85 @@
+package ua.artcode.week1;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.security.InvalidParameterException;
+
+public class FlashCardListTest {
```

Comment:
use assertThat in order to add more readability

Reply:
done